src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.AnyMobile.InvokeNativeHandler.cs

```diff
@@ -32,157 +31,237 @@ public partial class HttpClientHandler : HttpMessageHandler
 #else
 #error Unknown target
 #endif
+#pragma warning restore CA1823 // Unused field 'NativeHandlerType'
 
+        // UnsafeAccessor declarations for all native handler properties/methods
         private ICredentials? GetDefaultProxyCredentials()
-            => (ICredentials?)InvokeNativeHandlerGetter(() => Type.GetType(NativeHandlerType)!.GetMethod("get_DefaultProxyCredentials")!);
-
+        {
+            [UnsafeAccessor(UnsafeAccessorKind.Method, Name = "get_DefaultProxyCredentials")]
```

Comment:
Nit: I believe that the usual style is to have the local methods at the end of the method. (I do not have a strong opinion.)